ert_shared/cli/monitor.py

```diff
@@ -70,8 +71,20 @@ def monitor(self, tracker):
                 self._print_progress(event)
             if isinstance(event, EndEvent):
                 self._print_result(event.failed, event.failed_msg)
+                self._print_job_errors()
                 return
 
+    def _print_job_errors(self):
+        for snapshot_id, snapshot in self._snapshots.items():
+            for real_id, real in snapshot.get_reals().items():
+                for step_id, step in real.steps.items():
+                    for job_id, job in step.jobs.items():
+                        if job.status == state.JOB_STATE_FAILURE:
+                            print(
+                                f"Job failure in iteration: {snapshot_id} real: {real_id} "
+                                f"step: {step_id} job: {job_id} name: {job.name}\nError: {job.error}"
+                            )
+
```

Comment:
For large runs (500 is the new baseline), with e.g. 100 failing jobs, isn't this going to be a bit spammy? Also, considering the original report https://github.com/equinor/libres/issues/1108 it might be that simply reporting that 

```
Simulations failed with the following error: Simulation failed! All realizations failed!
39 of 73 realisations failed due to the `MAX_RUNTIME` setting—consider increasing it.
```

What do you think?

Reply:
Cli is spammy anyway, but we don't want to make it even more spammy:)